src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx

```diff
@@ -11,15 +11,16 @@ type StaticTooltipProps = {
 
 export default function StaticTooltip({ targetNode, children }: StaticTooltipProps) {
   const { value: enableLargeText } = useFeature("enableLargeText");
-  const { value: hitCounts } = useFeature("hitCounts");
+  const { value: hitCountsMode } = useStringPref("hitCounts");
 
   return ReactDOM.createPortal(
     <div
       className={classNames(
         "pointer-events-none absolute bottom-4 z-50 mb-0.5 flex translate-x-full transform flex-row space-x-px",
         enableLargeText && "bottom-6",
-        hitCounts ? "-right-[20px]" : "-right-1"
+        hitCountsMode === "show-counts" ? "-right-[20px]" : "-right-[10px]"
```

Comment:
I don't think we need this anymore, right? I think these styles would just be overridden by our inline style below.

Reply:
Yep, removed! I wasn't quick enough 😛